azure-quantum/azure/quantum/qiskit/backends/ionq.py

```diff
@@ -392,6 +430,15 @@ def __init__(self, name: str, provider: "AzureQuantumProvider", **kwargs):
             "configuration", default_config
         )
         super().__init__(configuration=configuration, provider=provider, **kwargs)
+    
+    def _azure_config(self) -> Dict[str, str]:
+        config = super()._azure_config()
+        config.update(
+            {
+                "is_default": True,
```

Comment:
Similarly to my other comment, I believe this is not correct and the configuration of this backend should not override the one from the parent class.

Reply:
Same for all the other backends that derive from `IonQBackend`.